openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -498,3 +498,10 @@ export const getRandomColor = (name: string) => {
 export const isUrlFriendlyName = (value: string) => {
   return !UrlEntityCharRegEx.test(value);
 };
+
+export const isSandboxOMD = () => {
+  return (
+    document.location.hostname === 'sandbox.open-metadata.org' ||
```

Comment:
@darth-coder00 , can we move out this hostname values to constant.

Reply:
Kept it here as it is going to be replaced by config from api